src/webfaf/templates/_helpers.html

```diff
@@ -371,11 +371,14 @@
 {%- endmacro %}
 
 {% macro external_urls(urls) -%}
-{% for url_obj in urls %}
-  <li><a
-    href="{{ url_obj.url }}">{{ url_obj.url }}
-    </a>
-  </li>
+{% for report_url in urls %}
+    {% for url_obj in report_url %}
+      <li><a
+        href="{{ url_obj.url }}">{{ url_obj.url }}
+        </a>
+      </li>
+    {% endfor %}
+    <li class="empty-line"></li>
```

Comment:
Maybe do something like this:
\<li class="empty-line"\> \<hr /\>  \</li\>

li.empty-line hr {margin: 0.5em 0;} 
li.empty-line:last-child { display: none; }

Reply:
hmm, this adds line. It stretches across the whole available space. I don't this it looks good. But I let it for others to decide. Ideas anyone?
![after](https://user-images.githubusercontent.com/12330670/33660046-d2ef63fc-da82-11e7-80ea-730cd4472e45.png)

Reply:
Yeah, it looks weird when the URLs are short. This is how it looks like with normal URLs. ![fafpr](https://user-images.githubusercontent.com/24898105/33662907-f6ebbcfa-da8d-11e7-820f-d279c89b7a7f.png)